config/initializers/smtp.rb

```diff
@@ -3,17 +3,26 @@
 
 if defined?(APP_CONFIG) && APP_CONFIG["email"]["smtp"]["enabled"]
   Portus::Application.config.action_mailer.delivery_method = :smtp
-
   smtp = APP_CONFIG["email"]["smtp"]
-  ActionMailer::Base.smtp_settings = {
+  smtp_settings = {
     address:              smtp["address"],
-    port:                 smtp["port"],
-    user_name:            smtp["user_name"],
-    password:             smtp["password"],
-    domain:               smtp["domain"],
-    authentication:       :login,
-    enable_starttls_auto: true
+    port:                 smtp["port"]
   }
+
+
+  if(!ENV['PORTUS_EMAIL_SMTP_AUTHENTICATION_DISABLED'])
```

Comment:
that being said, my change did apparently break things.  I should be able to update soon with that chagne as well.